README.md

```diff
@@ -31,7 +31,7 @@ Comlink turns this messaged-based API into a something more developer-friendly b
 
 ## Examples
 
-### [Running a simple function](https://github.com/GoogleChromeLabs/comlink/tree/master/docs/examples/01-simple-example)
+### [Running a simple function](./docs/examples/01-simple-example)
```

Comment:
This won’t work if you look at the README from the repo’s landing page (`https://github.com/GoogleChromeLabs/comlink`). I think full URLs work best here

Reply:
I was copying the existing link further down which seems to work fine:

```md
### [`SharedWorker`](./docs/examples/07-sharedworker-example)
```

Reply:
I didn’t catch that when I reviewed that, but it magically works. Amazing. never mind me then